lib/hanami/utils/attributes.rb

```diff
@@ -66,7 +66,16 @@ def initialize(hash = {})
       #   attributes[:unknown]      # => nil
       #   attributes['unknown']     # => nil
       def get(attribute)
-        @attributes[attribute.to_s]
+        value = @attributes
+
+        keys = attribute.to_s.split('.')
+        keys.each do |key|
+          break unless value
+
+          value = value[key]
+        end
```

Comment:
What do you think, maybe is better to get only last key from `split` instead unnecessary enumeration? I. e.:
```ruby
value = @attributes

key = attribute.to_s.split(?.).last
value = value[key] if value
```